lib/hammer_cli_foreman/image.rb

```diff
@@ -31,6 +31,7 @@ class ListCommand < HammerCLIForeman::ListCommand
         field nil, _("Operating System"), Fields::SingleReference, :key => :operatingsystem
         field :username, _("Username")
         field :uuid, _("UUID")
+        field :user_data,  _("User Data")
```

Comment:
is this relevant for all compute resources? I know the oVirt support it, but I'm not sure that all the other compute resource supports it too

Reply:
Yes, including GCE I can see that all default list of compute resources have this `user_data` field.